src/libstore/unix/user-lock.cc

```diff
@@ -74,8 +74,13 @@ struct SimpleUserLock : UserLock
             debug("trying user '%s'", i);
 
             struct passwd * pw = getpwnam(i.c_str());
-            if (!pw)
-                throw Error("the user '%s' in the group '%s' does not exist", i, settings.buildUsersGroup);
+            if (!pw) {
+                printError("error: the user '%s' in the group '%s' does not exist", i, settings.buildUsersGroup);
+#if __APPLE__
+                printError("note: If you are using Sequoia, you may need to run this script: 'https://github.com/NixOS/nix/blob/master/scripts/sequoia-nixbld-user-migration.sh'");
```

Comment:
```suggestion
                printError("note: If you are using Sequoia or newer, you may need to run this script: 'https://github.com/NixOS/nix/blob/master/scripts/sequoia-nixbld-user-migration.sh'");
```

Reply:
Or better, say "macOS 15.0 or newer" for people who don't have macOS codenames memorized.

Reply:
```suggestion
                printError("note: If you are using macOS 15.0 Sequoia or newer, you may need to run this script: 'https://github.com/NixOS/nix/blob/master/scripts/sequoia-nixbld-user-migration.sh'");
```

Or better, for people who don't have macOS version numbers memorized ;)